typeql-src/modules/ROOT/pages/schema/type-definitions.adoc

```diff
@@ -17,15 +17,15 @@ See the table below for the most simple statements that can be used for type def
 
 | <<_subtyping,Subtyping>> entity type
 | `<label> sub <type>;`
-| `person sub user;`
+| `employee sub person;`
 
 | <<_subtyping,Subtyping>> relation type
 | `<label> sub <type>;`
-| `person sub user;`
+| `hetero_marriage sub marriage;`
```

Comment:
```suggestion
| `car-ownership sub ownership;`
```

It's a bit more neutral example, and we do use kebab-case (substitute whitespace by hyphens).